test/OpenTelemetry.Exporter.InfluxDB.Tests/Utils/MeterProviderBuilderTestExtensions.cs

```diff
@@ -0,0 +1,32 @@
+// <copyright file="MeterProviderBuilderTestExtensions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Resources;
+
+namespace OpenTelemetry.Exporter.InfluxDB.Tests.Utils;
+
+public static class MeterProviderBuilderTestExtensions
+{
+    public static MeterProviderBuilder ConfigureDefaultTestResource(this MeterProviderBuilder meterProviderBuilder)
+    {
+        return meterProviderBuilder.ConfigureResource(builder => builder.AddService(
+            serviceName: "my-service",
```

Comment:
What happened if you add resource with the same key as for regular tag?
How it is handled in collector influxdb exporter?

Reply:
Resources take precedence over regular tags, the same behavior as [here](https://github.com/influxdata/influxdb-observability/blob/main/otel2influx/metrics_telegraf_prometheus_v1.go#L38).

Reply:
I would cover this case in one test case.

Reply:
Could be done as a separate PR?

Reply:
Sure.